Makefile

```diff
@@ -1,12 +1,27 @@
-push: output ldoc changelogs manpages
-	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s --exclude src html/ awesome.naquadah.org:/var/www/awesome.naquadah.org/
-	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s --delete src/build/doc/ awesome.naquadah.org:/var/www/awesome.naquadah.org/doc/api
-	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s /usr/share/asciidoc/icons awesome.naquadah.org:/var/www/awesome.naquadah.org/doc/manpages/icons
+all: output ldoc changelogs manpages
+
+build: all
+	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s --exclude src html/ build
+	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s --delete src/build/doc/ build/doc/api
+	rsync -PaOvz --chmod=u=rwX,g=rwX,o=rX,Dg+s /usr/share/asciidoc/icons build/doc/manpages/icons
+
+update_from_travis: export HOME=/dev/null
+update_from_travis: build
+	GIT_AUTHOR_NAME="awesome-robot on Travis CI" \
+		GIT_AUTHOR_EMAIL="awesome-robot@users.noreply.github.com" \
+		GIT_COMMITTER_NAME="$$GIT_AUTHOR_NAME" \
+		GIT_COMMITTER_EMAIL="$$GIT_AUTHOR_EMAIL" \
+		cd build \
+		&& git add --all . \
+		&& git commit -m \
+			"Update from Travis for awesome-www@$$(cd .. && git rev-parse --short HEAD)" \
+		&& git push
```

Comment:
Because this depends on `build` which itself depends on `all`, it causes the wiki to be rebuilt again. Perhaps this part of the build is better done in `.travis.yml`?
Also, setting the env vars like this only applies them to the `cd`-command, so the git commit failed due to missing author info.